docs/data/sources/google-ads.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Google Ads (Beta)
+description: Amplitude Data's Google Ads integration lets you import your Google Ad spend, click and impression data with just a few clicks.
+---
+
+Amplitude Data's Google Ads integration lets you import your Google Ad spend, click and impression data with just a few clicks.
+
+--8<-- "includes/closed-beta.md"
+
+## Setup
+
+### Prerequisites
+
+To set up, you need the following: 
+
+- `Google Ads Customer ID` of the ad account you would like imported (https://support.google.com/google-ads/answer/1704344?hl=en)
+
+### Amplitude setup 
+
+1. In Amplitude, navigate to **Data Sources**, then find **Google Ad** in the **I want to import data into Amplitude** tab.
+
+![Google Add Source](../../assets/images/marketing-analytics/add-sources.png)
+
+!!! note 
+    This integration must be enabled on a per-project basis.
+
+2. Log into Google and grant Amplitude permission in the consent form.
+![Google Login Image](../../assets/images/marketing-analytics/google-login.png)
+3. Enter the `Google Ads Customer ID` for all the ad accounts you would like to sync the data.
```

Comment:
"for all the ad accounts" seems misleading here, since it's just one. Just "for the ad account you would like to sync data from."?